global-script.lua

```diff
@@ -3212,9 +3208,9 @@ function onSwapButtonClicked(target_obj, source_color, alt_click)
     source_obj = playerBlocks[source_color]
 
     if not target_color then
-        player.print("That seat is already taken.")
+        broadcastToColor("That seat is already taken.", source_color)
     elseif not source_obj then
-        player.print("You are already seated.")
+        broadcastToColor("You have already chosen a spirit.", source_color)
     elseif target_color == source_color then
         -- player.print("Okay, you trade places with yourself.")
```

Comment:
if you're going to be keeping this comment around, it should probably be updated to new function call

Reply:
swapPlayerAreas actually really ought to be doing this check itself, so moved it there (and removed the comment accordingly.)